scout/constants/__init__.py

```diff
@@ -183,46 +183,51 @@
 
 CALLERS = {
     "snv": [
-        {"id": "gatk", "name": "GATK"},
-        {"id": "freebayes", "name": "Freebayes"},
-        {"id": "samtools", "name": "SAMtools"},
         {"id": "bcftools", "name": "Bcftools"},
         {"id": "deepvariant", "name": "DeepVariant"},
+        {"id": "freebayes", "name": "Freebayes"},
```

Comment:
Doesn't have to be from this PR, but since there are several repetitions we could set some constants to use in these lists

Reply:
Sure, there are only a handful of them, but why not. I'll also move this deeper down the constants tree.